### PR TITLE
Some small fixes

### LINKED ISSUE #177: Disambiguate set_name (again)
```
    |                 ^^^^^^^^ multiple `set_name` found
    |
    = note: candidate #1 is defined in an impl of the trait `gtk4::EventControllerExt` for the type `O`
    = note: candidate #2 is defined in an impl of the trait `gtk4::WidgetExt` for the type `O`
    = note: candidate #3 is defined in an impl of the trait `gtk4::prelude::WidgetExtManual` for the type `O`
```

### PR DESCRIPTION
**Pull request by sophie-h**
Fixes #177

### CODE REVIEW

File: gtk4/Gir.toml [hunk 1]
```diff
@@ -2350,16 +2350,6 @@ manual_traits = ["WidgetExtManual"]
     doc_trait_name = "WidgetExtManual"
     # remove after https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2887
     [[object.function]]
-    name = "get_css_name"
-        [object.function.return]
-        nullable = true
-    # remove after https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2887
-    [[object.function]]
-    name = "get_name"
```
Comment (sdroege) [inline gtk4/Gir.toml:2358]: Can't this be renamed? I thought we added a gir configuration for that in the meantime @GuillaumeGomez ?

Comment (GuillaumeGomez) [reply gtk4/Gir.toml:2358]: Euh... Maybe? Sorry, I don't remember at all...

Comment (sophie-h) [reply gtk4/Gir.toml:2358]: `rename =` works

**Comment by sophie-h** [comment]
Draft until https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2905#note_1017989 is clarified.

**Comment by sophie-h** [comment]
> Draft until https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2905#note_1017989 is clarified.

Upstream seems to agree with me. Ready for review

File: gtk4/src/auto/widget.rs [hunk 2]
```diff
@@ -59,6 +59,13 @@ impl Widget {
     }
 }
 
+impl fmt::Display for Widget {
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/widget.rs:62]: Is this "automatically" generated code? if not, it should be moved to src/widget.rs

**Comment by bilelmoussaoui** [comment]
Do you think you can have a look at gtk::Entry as well? it still has the weird get_text returning a `Option<GString>` otherwise looks good to me

File: gtk4/src/auto/widget.rs (hunk 2 above)
Comment (sophie-h) [reply gtk4/src/auto/widget.rs:62]: This is generated.

**Comment by sophie-h** [comment]
> Do you think you can have a look at gtk::Entry as well? it still has the weird get_text returning a `Option<GString>` otherwise looks good to me

This should be provided by gtk::Editable and that's fixed with this PR :)

**Comment by bilelmoussaoui** [comment]
Ah my bad. Looks good to me then, thanks!

File: gtk4/src/auto/widget.rs (hunk 2 above)
Comment (bilelmoussaoui) [reply gtk4/src/auto/widget.rs:62]: It weird how gir moved it to the top of the file instead of the bottom, which confused me :)

**Pull request merged**